lumicks/pylake/force_calibration/calibration_models.py

```diff
@@ -99,3 +102,135 @@ def calibration_results(self, fc, diffusion_constant):
+           tweezers. Comput. Phys. Commun. 175, 572–573 (2006).
+    .. [4] Berg-Sørensen, K., Peterman, E. J. G., Weber, T., Schmidt, C. F. & Flyvbjerg, H. Power
+           spectrum analysis for optical tweezers. II: Laser wavelength dependence of parasitic
+           filtering, and how to achieve high bandwidth. Rev. Sci. Instrum. 77, 063106 (2006).
+    .. [5] Tolić-Nørrelykke, S. F, and Flyvbjerg, H, "Power spectrum analysis with least-squares
+           fitting: amplitude bias and its elimination, with application to optical tweezers and
+           atomic force microscope cantilevers." Review of Scientific Instruments 81.7 (2010)
+    .. [6] Tolić-Nørrelykke S. F, Schäffer E, Howard J, Pavone F. S, Jülicher F and Flyvbjerg, H.
+           Calibration of optical tweezers with positional detection in the back focal plane,
+           Review of scientific instruments 77, 103101 (2006).
+    """
+
+    def __name__(self):
+        return "ActiveCalibrationModel"
+
+    def __init__(
+        self,
+        driving_data,
+        force_voltage_data,
+        sample_rate,
+        bead_diameter,
+        driving_frequency_guess,
+        viscosity=1.002e-3,
+        temperature=20,
+        num_windows=5,
+    ):
+        """
+        Active Calibration Model.
+
+        This model fits data acquired in the presence of nanostage or mirror oscillation.
+
+        Parameters
+        ----------
+        driving_data : array_like
+            Array of driving data.
+        force_voltage_data : array_like
+            Uncalibrated force data in volts.
+        sample_rate : float
+            Sample rate at which the signals we acquired.
+        bead_diameter : float
+            Bead diameter [um].
+        driving_frequency_guess : float
+            Guess of the driving frequency.
+        viscosity : float, optional
+            Liquid viscosity [Pa*s].
+        temperature : float, optional
+            Liquid temperature [Celsius].
+        num_windows : int, optional
+            Number of windows to average for the uncalibrated force. Using a larger number of
+            windows potentially increases the bleed, but may be useful when the SNR is low.
+        """
+        super().__init__(bead_diameter)
+        self.driving_frequency_guess = driving_frequency_guess
+        self.sample_rate = sample_rate
+        self.viscosity = viscosity
+        self.temperature = temperature
+        self.num_windows = num_windows
+
+        # Estimate driving input and response
+        amplitude_um, self.driving_frequency = estimate_driving_input_parameters(
+            sample_rate, driving_data, driving_frequency_guess
+        )
+        self.driving_amplitude = amplitude_um * 1e-6
+        self.response_power_density, self.frequency_resolution = response_power_density(
+            force_voltage_data, sample_rate, self.driving_frequency, num_windows
+        )
```

Comment:
It's a good point.

While documenting it, I actually realized that these properties should probably remain private (at least for now), so I've made them private. Nevertheless, it seems they need some additional documentation.

Power density is a density. Power is the power density integrated over the frequency axis.

The maximum of the peak here is indeed a density (Volt^2 / Hz). To get the actual power you have to integrate it over the bin still. I've also renamed `frequency_resolution` to `__frequency_bin_width` since it is more descriptive. Have a look whether you think the new names plus their docstrings work for you.